src/transformers/generation/utils.py

```diff
@@ -156,7 +156,7 @@ class GenerateDecoderOnlyOutput(ModelOutput):
             the model's documentation. Usually, a [`~cache_utils.Cache`] instance.
     """
 
-    sequences: torch.LongTensor = None
+    sequences: Optional[torch.LongTensor] = None
```

Comment:
```suggestion
    sequences: torch.LongTensor
```
The correct solution here is to remove the default